integration/cli/__snapshots__/yarn-created.json

```diff
@@ -0,0 +1,54 @@
+[
+  ".gitignore",
+  ".vscode",
+  ".vscode/extensions.json",
+  ".vscode/launch.json",
+  ".vscode/settings.json",
+  "README.md",
+  "astro.config.ts",
+  "icons",
+  "icons/languages",
+  "icons/languages/css.svg",
+  "icons/languages/html.svg",
+  "icons/languages/js.svg",
+  "icons/languages/json.svg",
+  "icons/languages/markdown.svg",
+  "icons/languages/sass.svg",
+  "icons/languages/ts.svg",
+  "package.json",
+  "public",
+  "public/favicon.svg",
+  "public/logo-dark.svg",
+  "public/logo.svg",
+  "src",
+  "src/content",
+  "src/content/config.ts",
+  "src/content/tutorial",
+  "src/content/tutorial/1-basics",
+  "src/content/tutorial/1-basics/1-introduction",
+  "src/content/tutorial/1-basics/1-introduction/1-welcome",
+  "src/content/tutorial/1-basics/1-introduction/1-welcome/_files",
+  "src/content/tutorial/1-basics/1-introduction/1-welcome/_files/counter.js",
+  "src/content/tutorial/1-basics/1-introduction/1-welcome/_solution",
+  "src/content/tutorial/1-basics/1-introduction/1-welcome/_solution/counter.js",
+  "src/content/tutorial/1-basics/1-introduction/1-welcome/content.md",
+  "src/content/tutorial/1-basics/1-introduction/meta.md",
+  "src/content/tutorial/1-basics/meta.md",
+  "src/content/tutorial/meta.md",
+  "src/env.d.ts",
+  "src/templates",
+  "src/templates/default",
+  "src/templates/default/.gitignore",
+  "src/templates/default/counter.js",
+  "src/templates/default/index.html",
+  "src/templates/default/javascript.svg",
+  "src/templates/default/main.js",
+  "src/templates/default/package-lock.json",
+  "src/templates/default/package.json",
+  "src/templates/default/public",
+  "src/templates/default/public/vite.svg",
+  "src/templates/default/style.css",
+  "tsconfig.json",
+  "uno.config.ts",
+  "yarn.lock"
+]
```

Comment:
Newline missing.

Reply:
This is written by the snapshot itself. Unless we can modify the generator to write this line, I would be strongly against manually updating it every single time.

Reply:
Ah yea that's true. Changing automatically generated files isn't ideal so totally agree. Kind of annoying that it doesn't add the newline there.

Reply:
Yeah I agree, don't like it either.